#
# This patch makes the magnifier tool use nearest filtering to avoid
# getting a blurry zoomed image. Useful if you need a more accurate zoom.
#
--- src/mag/mag.c
+++ src/mag/mag.c
@@ -1111,8 +1111,8 @@
     glBindTexture (ms->target, ms->texture);
 
     /* Load the parameters */
-    glTexParameteri (ms->target, GL_TEXTURE_MIN_FILTER, GL_LINEAR);
-    glTexParameteri (ms->target, GL_TEXTURE_MAG_FILTER, GL_LINEAR);
+    glTexParameteri (ms->target, GL_TEXTURE_MIN_FILTER, GL_NEAREST);
+    glTexParameteri (ms->target, GL_TEXTURE_MAG_FILTER, GL_NEAREST);
     glTexParameteri (ms->target, GL_TEXTURE_WRAP_S, GL_CLAMP);
     glTexParameteri (ms->target, GL_TEXTURE_WRAP_T, GL_CLAMP);
 

